transmart-rest-api-e2e/src/test/groovy/tests/rest/v2/DataExportSpec.groovy

```diff
@@ -552,6 +553,12 @@ class DataExportSpec extends RESTSpec {
         return [jobName: jobId, downloadResponse: downloadResponse]
     }
 
+    /**
+     * This test fails randomly, because sometimes job runs too fast and it is already completed
+     * while the test is trying to cancel it
+     * Fix will be covered by: https://jira.thehyve.nl/browse/TMT-301
+     */
+    @Ignore
```

Comment:
Alternative approach here: https://github.com/thehyve/transmart-core/pull/263/files#diff-d7d988d63ef259059b18505777b79275L574

Reply:
ok, looks like a good solution, I will wait till [your PR](https://github.com/thehyve/transmart-core/pull/263) is merged and rebase this one on it (remove the comment).